prdoc/pr_6221.prdoc

```diff
@@ -0,0 +1,10 @@
+title: Enable approval-voting-parallel by default on kusama
```

Comment:
Probably I am missing some context but looks like this title is not connected neither to PR nor to commit description.

Reply:
Good catch! 👍

Reply:
```suggestion
title: "snowbridge: allow account conversion for Ethereum accounts"
```